.ldrelease/clone-push-bitbucket.sh

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
-
-# This is expected to run after update-bitbucket.sh has updated the verison information.
-set -ev
-
-mkdir -p metadataUpdates
-git clone git@bitbucket.org:launchdarkly/ld-find-code-refs-pipe.git metadataUpdates/bitbucket
-cp build/metadata/bitbucket/* metadataUpdates/bitbucket/
-cd metadataUpdates/bitbucket
-git add -u
-git commit -m "Update version"
-git push origin master
```

Comment:
This file is not deleted but renamed to `publish-bitbucket-metadata.sh` to be consistent with the releaser step where it's invoked.